src/ui/components/Scanner/Scanner.scss

```diff
@@ -44,6 +44,19 @@
     }
   }
 
+  .actions-button {
+    position: absolute;
+    bottom: 2.25rem;
+    left: 50%;
+    transform: translateX(-50%);
+
+    .paste-mkid-btn {
+      width: calc(100vw - 2.5rem);
+      --border-radius: 2rem;
+      --background: #f6f6f680;
```

Comment:
1. No need to use hardcoded values for colors.

2. No need to even use a new component for this button, you can just use the `PageFooter` and pass a `secondary-button` like I showed you in [this PR](https://github.com/cardano-foundation/cf-identity-wallet/pull/451/files). Will add more comments on this below.